.travis.yml

```diff
@@ -1,8 +1,10 @@
 sudo: false
 language: erlang
 otp_release:
-  - 21.1
-  - 20.0
+  - 23.0
+  - 22.3
+  - 21.3
+  - 20.3
   - 19.3
```

Comment:
We can totally drop support for `19.3`. I would even argue that we can drop support for `20.3`, too.

Reply:
😄 I wouldn't, though, but if you prefer to remove it from `.travis.yml` alone (e.g. not mess with `minimum_otp_vsn` in `rebar.config`), sure, you're the bawss.

**Note**: I like to keep my "versions" with the same schedule as `rebar3` (so it's 19+, for now).

Reply:
Our policy (when Inaka was a company) was to support only the last 2 versions of OTP. If you work with older versions of OTP, use older releases of our libs. And if you need new features in them, just ask us… and we backport them.

Reply:
We backported stuff a total of… one single time through our whole 7 years of existence. 🤷

Reply:
I understand you don't want to support 19.3 or even more recent versions (not everybody has the luxury of constantly moving a few hundred libs **up** all the time - and when the deps go deep, it becomes a real hassle).

Let me know, so I have to **not** care. And tell me explicitly which ones you want to support, so I know.

Reply:
I would be pragmatic here: leave all there until we make a change that breaks the old ones. If/when that's the case, instead of forcing ourselves to write backwards-compatible code, just increase minimum_otp_vsn in rebar.config.

The perfect example of this are stack traces. I would not support `erlang:get_stacktrace()` and `Kind:Reason:Stack` in the same codebase. Once we move forward... We entirely remove support for the previous stuff.

Reply:
But this is @ferigis' repo. He gets to decide what to do here, not me.

Reply:
> leave all there

Exactly what I was suggesting. Don't bump for the sake of bumping (though I do understand there's an "implied" cost in maintenance), and keep testing until something needs to break in a given past version, out of necessity.

Reply:
> The perfect example of this are stack traces. I would not support erlang:get_stacktrace() and Kind:Reason:Stack in the same codebase. Once we move forward... We entirely remove support for the previous stuff.

Sure, but until you don't, you'll get warnings (which some might not find acceptable, in their context, so have to fork or stay in a previous version), and when you do it might be very abrupt, since you're relying on OTP to not be abrupt.

In any case, it's always a maitainer's choice and can be imposed via `rebar.config`'s `minimum_otp_vsn`, for example, so it's clear.